operator/src/test/java/org/keycloak/operator/KeycloakDeploymentE2EIT.java

```diff
@@ -246,4 +250,46 @@ public void testHostnameStrictDisabled() {
         }
     }
 
+    // Reference curl command:
+    // curl --insecure --data "grant_type=password&client_id=admin-cli&username=admin&password=adminPassword" https://localhost:8443/realms/master/protocol/openid-connect/token
+    @Test
+    public void testInitialAdminUser() {
```

Comment:
Please add a test for checking that nothing happens when there's a mismatch between the secret and  (already) created admin user in DB. This typically happens when the user changes the initial password and deployment is later (due to some other reason) restarted.

Reply:
I added a "step" in the current test to verify also this situation, can you check that this is a reasonable compromise @vmuzikar ?

Reply:
👍